### PR TITLE
fix: project grants

### LINKED ISSUE #4004: Too many events on project grant audit
**Describe the bug**
The change log on a project grant shows to many changes. It looks like there are all events from the whole project shown and not only of the project grant.

**To Reproduce**
1. Go to a projec grant in the console
2. See last changes
3. To many events

**Expected behavior**
Should only show events from the project grant

**Screenshots**
image

### PR DESCRIPTION
**Pull request by livio-a**
closes #4004

### CODE REVIEW

**Review by peintnermax — DISMISSED** [review]
Frontend looks good!

File: internal/query/user_membership.go [hunk 1]
```diff
@@ -294,6 +308,7 @@ func prepareOrgMember() string {
 		"NULL::STRING AS "+membershipIAMID.name,
 		"NULL::STRING AS "+membershipProjectID.name,
 		"NULL::STRING AS "+membershipGrantID.name,
+		//"NULL::STRING AS "+membershipGrantGrantedOrgID.name,
```
Comment (hifabienne) [inline internal/query/user_membership.go:311]:
```suggestion
```

File: internal/query/user_membership.go [hunk 2]
```diff
@@ -311,6 +326,7 @@ func prepareIAMMember() string {
 		InstanceMemberIAMID.identifier(),
 		"NULL::STRING AS "+membershipProjectID.name,
 		"NULL::STRING AS "+membershipGrantID.name,
+		//"NULL::STRING AS "+membershipGrantGrantedOrgID.name,
```
Comment (hifabienne) [inline internal/query/user_membership.go:329]:
```suggestion
```

File: internal/query/user_membership.go [hunk 3]
```diff
@@ -328,6 +344,7 @@ func prepareProjectMember() string {
 		"NULL::STRING AS "+membershipIAMID.name,
 		ProjectMemberProjectID.identifier(),
 		"NULL::STRING AS "+membershipGrantID.name,
+		//"NULL::STRING AS "+membershipGrantGrantedOrgID.name,
```
Comment (hifabienne) [inline internal/query/user_membership.go:347]:
```suggestion
```

File: internal/query/user_membership.go [hunk 4]
```diff
@@ -346,7 +363,10 @@ func prepareProjectGrantMember() string {
 		"NULL::STRING AS "+membershipIAMID.name,
 		ProjectGrantMemberProjectID.identifier(),
 		ProjectGrantMemberGrantID.identifier(),
-	).From(projectGrantMemberTable.identifier()).MustSql()
+		//ProjectGrantColumnGrantedOrgID.identifier(),
+	).From(projectGrantMemberTable.identifier()).
+		//LeftJoin(join(ProjectGrantColumnGrantID, ProjectGrantMemberGrantID)).
+		MustSql()
```
Comment (hifabienne) [inline internal/query/user_membership.go:369]:
```suggestion
	).From(projectGrantMemberTable.identifier()).
		MustSql()
```

**Review by hifabienne — APPROVED** [review]

**Pull request merged**